code/style.css

```diff
@@ -1,7 +1,551 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+html,
+body {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: white;
+  color: black;   
+}
+
+* {
+  box-sizing: border-box;
+  position: relative;
+}
+
+h1  {
+  text-align: center;
+  margin: 2rem 0;
+  font-size: 7vw;
+  font-family: 'Quicksand', sans-serif;
+  font-weight: bolder;
+}
+
+h2 {
+  font-family: 'Quicksand', sans-serif;
+  font-size: 3vw;
+}
+
+a {
+  font-family: 'Quicksand', sans-serif;
+}
```

Comment:
Again really nice structured 🏆
If you don't wanna repeat the font-family you can always put it in the body styling.